modules/arrow/src/types.d.ts

```diff
@@ -0,0 +1,18 @@
+type TypedIntArray =
+  Int8Array | Uint8Array |
+  Int16Array | Uint16Array |
+  Int32Array | Uint32Array |
+  Int32Array | Uint32Array;
+
+type TypedFloatArray =
+    Float32Array |
+    Float64Array;
+
+type TypedArray = TypedIntArray | TypedFloatArray;
+
+export type AnyArrayType = Array<any> | TypedIntArray | TypedFloatArray;
```

Comment:
Nit: This is fine, but I wonder if Arrow doesn't already have something like this - since we are importing arrow.  They probably handle BigInt arrays as well

Reply:
Typing is there but is not exported

Reply:
I found it in another file: https://github.com/apache/arrow/blob/c08054867d08231a160288f14cd69216b46b221c/js/src/interfaces.ts#L26
I'll import it instead of defining here.

Reply:
@trxcllnt Any recommendations for how to import such helper types from arrow js?